h2o-py/h2o/frame.py

```diff
@@ -2564,6 +2567,88 @@ def isax(self, num_words, max_cardinality, optimize_card=False, **kwargs):
         if max_cardinality <= 0: raise H2OValueError("max_cardinality must be greater than 0")
         return H2OFrame._expr(expr=ExprNode("isax", self, num_words, max_cardinality, optimize_card))
 
+    def convert_H2OFrame_2_DMatrix(self, predictors, yresp, h2oXGBoostModel):
+        '''
+        This method will convert a H2OFrame to a DMatrix that can be used by native XGBoost.  The H2OFrame contains
+        numerical and enum columns alone.  Note that H2O one-hot-encoding introduces a missing(NA)
+        column. There can be NAs in any columns.
+
+        Follow the following steps to compare H2OXGBoost and native XGBoost:
+
+        1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction
```

Comment:
same comments as above. End sentences with colon or period. Uppercase first word of steps.